pkg/network/encoding/format.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	model "github.com/DataDog/agent-payload/v5/process"
 	"github.com/DataDog/datadog-agent/pkg/network"
+	"github.com/DataDog/datadog-agent/pkg/process/protocols/telemetry"
```

Comment:
Is this package in `main`?

Reply:
It is, but I messed up with the path. Fixed!